_brms-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit 66df6d3774cf1c1e978a23ef0640d3e65d9688e0
+Subproject commit 7fb9b53309204cd3d2040306a5bfdbd95706ac63
```

Comment:
Looks like you are reverting the QS from BxMS-6.1.0.GA to BRMS-6.0.2.GA. Notice: https://github.com/jboss-developer/jboss-brms-quickstarts/releases.

I suggest you drop this change.